autograder/submitty_autograding_shipper.py

```diff
@@ -37,6 +37,58 @@
 JOB_ID = '~SHIP~'
 
 
+# This function is copied, would rather use the version in this file instead.
```

Comment:
@MasterOdin
If you tell me how I should properly refer to the python function in the other directory, I will update this code to remove repeated code.  What I was trying wasn't working.  Thanks.

Reply:
I would recommend moving this to `submitty_utils` as the point of this library was to make it easy to share code between various discrete components.

Reply:
Thanks.  I believe I've got it as you suggest and there is no more code duplication - yay